bundles/org.openhab.binding.avmfritz/src/test/java/org/openhab/binding/avmfritz/internal/ahamodel/AVMFritzDeviceListModelTest.java

```diff
@@ -479,12 +478,11 @@ public void validateSwitchGroupModel() {
         assertEquals("17,18", group.getGroupinfo().getMembers());
     }
 
-    @Deprecated
-    private Optional<AVMFritzBaseModel> findModel(@NonNull String name) {
+    private Optional<AVMFritzBaseModel> findModel(String name) {
```

Comment:
Did you remove the null-annotation deliberately?

Reply:
Yes. According to our guidelines we should not use it. Does it make sense in a unit test?